### DISCUSSION TITLE
How to setup Telegram channel

### DISCUSSION THREAD

**Discussion by Turab** [chatwoot/chatwoot#2353] [Self hosted]
I am interested in Chatwoot for some couple of months. Chatwoot has been advertising in its website that it supports Telegram and I can see that when you want to create a channel, Telegram is on the list. BUT, I don't see ANY documentation about it. Telegram is grayed out in channel list; you can never create one. The more interesting thing is, no one is talking about it.. What, how, why?

**Comment by iammattmartin** [comment]
It's worth searching....

#1843
